jni/patches/faiss/0001-Custom-patch-to-support-multi-vector.patch

```diff
@@ -0,0 +1,221 @@
+From 864c1abe7bdced5d306e871ea2bd73e1e35987fd Mon Sep 17 00:00:00 2001
```

Comment:
Can we add a README about patches? Or update DEV guide? Want to make sure we know what we are doing on lib upgrades.

Reply:
Let me add in DEV guide on how to generate patches or how to update patches.